### ISSUE TITLE
No Bluetooth or No Legacy Mode

### ISSUE THREAD

**Issue by VincentVintage** [AndroidAudioMods/ViPER4Android#44]
Hello, I cant get Youtube and Bluetooth working the same Time.

I'm on Pixel 6 latest Stable A13 of May 2023, rooted.
So when I'm using Sony Linkbuds S with LDAC on I got following behaviour:

Legacy Mode Off:
Spotify works
Youtube Not

Legacy On:
Spotify works not
Youtube works not

Without Bluetooth using Phone Speakers I got this:

Legacy Mode Off:
Spotify works
Youtube not

Legacy On:
Spotify works
Youtube works!

So basically there is no combination where I can get Youtube on Bluetooth working

**Comment by tekq** [comment]
Having a similar issue on the OnePlus 7T Pro (using YAAP and rooted) and using Sony WH-1000XM4, but in my case Tidal doesn't work

**Comment by iscle** [comment]
Interesting... Please share a screenshot of the status dialog when this happens in order to see what's going on :)

**Comment by tekq** [comment]
> Interesting... Please share a screenshot of the status dialog when this happens in order to see what's going on :)

Here's the screenshots of the status dialog https://imgur.com/a/JKOIb1D

Edit: I managed to get it working. If I play something, and then connect the headphones then it will work properly

**Comment by erqaw84** [comment]
I'm having a similar issue. While using Bluetooth headphones, Processing and Enabled indicate Yes while nothing is playing. Both change to No when music is playing. 

> Edit: I managed to get it working. If I play something, and then connect the headphones then it will work properly

This also workes for me but just temporarily, the problem goes back after music is paused or a song ends.

I hope the problem will be solved, thank you! :)
